tox.ini

```
@@ -1,5 +1,5 @@
 [tox]
-envlist = py38-celery{50}-django{32,42},quality,docs
+envlist = py{38,311,312}-celery{50}-django{42},quality,docs
```

env name should be updated to match test env requirments pattern.
```suggestion
envlist = py{38,311,312}-celery{53}-django{42},quality,docs
```